checker-qual/src/main/java/org/checkerframework/framework/qual/DefaultQualifier.java

```diff
@@ -63,6 +66,13 @@
      */
     TypeUseLocation[] locations() default {TypeUseLocation.ALL};
 
+    /**
+     * When used on a package, whether the defaults should also apply to subpackages.
+     *
+     * @return whether the default should be inherited by subpackages
+     */
+    boolean applyToSubpackages() default true;
```

Comment:
As for naming, I can't recall offhand whether there's a good precedent between the style "applyToSubpackages" and the style "appliesToSubpackages." Both seem workable. I suspect that I would have picked "appliesToSubpackages" if I were writing this from scratch, but I don't have much justification for that :)